src/applications/discharge-wizard/helpers/index.jsx

```diff
@@ -263,12 +263,15 @@ export const deriveIsAirForceAFRBAPortal = formValues =>
 
 // v2 Helpers
 
+// Changes Marine Corps discharge board to Navy for naming on results pages.
 export const determineBranchOfService = key =>
   key === RESPONSES.MARINE_CORPS ? RESPONSES.NAVY : key;
 
+// Determines old discharges
 export const determineOldDischarge = (dischargeYear, dischargeMonth) =>
   differenceInYears(new Date(), new Date(dischargeMonth, dischargeYear)) >= 15;
 
+// Determines the label used on the review page.
```

Comment:
Consider: maybe a bit more specific here? Since this utility creates a full-sentence version of some short answers like "I dunno," we could call out that it makes it more clear for review. Don't stress about addressing this if you disagree; just a note.